benchmark/single-source/DataBenchmarks.swift

```diff
@@ -13,638 +13,345 @@
 import TestsUtils
 import Foundation
 
+let d: [BenchmarkCategory] =  [.validation, .api, .Data]
+
+/// Workload scaling: Inner loop multipliers
+let M = 10_000  /// default
```

Comment:
The only quibble I have here is that these could use a bit better set of names.

Reply:
We should probably "precompute and inline" them all. I had extracted these constants only to demonstrate how's the `legacyFactor` derived. 

Would you be OK with having the multipliers all inline, or do you have a suggestion for better names for the constants?

Reply:
Going with inlined constants, unless you wan't me to revert back to constants.